savePositionData.js

```diff
@@ -44,6 +44,10 @@ async function saveOrValidateInitPositionInfo(position) {
     return;
   }
 
+  //logger.note("Saved initial position data to SQL", position.id);
+}
+
+const saveNewPositionInDB = async (position, initData, historicalData) => {
```

Comment:
do not save position without its historical data

Reply:
removed